src/include/miopen/convolution.hpp

```diff
@@ -73,6 +73,27 @@ using ExtraKernelArgs = std::tuple<int /*N*/,
 struct ConvFwdTensors;
 struct ConvWrwTensors;
 
+enum class ConvDirection
+{
+    Fwd = 1,
+    Bwd = 2,
+    WrW = 4
+};
+
+void LogCmdConvolution(const miopenTensorDescriptor_t& xDesc,
+                       const miopenTensorDescriptor_t& wDesc,
+                       const miopenConvolutionDescriptor_t& convDesc,
+                       const miopenTensorDescriptor_t& yDesc,
+                       const ConvDirection& conv_dir,
+                       bool is_immediate);
+
+void LogCmdFindConvolution(const miopenTensorDescriptor_t& xDesc,
```

Comment:
These global declarations are used only in tests. Can you please copy declarations directly into tests and remove them from the public header?

Reply:
Actually `LogCmdConvolution` and `LogCmdFindConvolution` are used in [convolution_api.cpp](https://github.com/ROCmSoftwarePlatform/MIOpen/blob/LWPTUNA-63/src/convolution_api.cpp#L420). I had to place them in header file so that I could test them. I was not sure where would be the best header file to place for these function declaration. So, I just placed it inside convolution.hpp. I am open to suggestions :)

Reply:
@bghimireamd 
> I am open to suggestions :)

Here is the suggestion: 
> Can you please copy declarations directly into tests and remove them from the public header?

Reply:
You insert copies of declarations of `LogCmdFindConvolution` and `LogCmdConvolution` into `test/gtest/log_test_helper.hpp`

Reply:
modified code as per request.